src/utils.ts

```diff
@@ -74,6 +75,10 @@ export function isDefaultNotificationDisabled() : boolean | undefined {
 	return notificationSetting;
 }
 
+export function getOpenAtStartupSetting() : boolean {
+	return vscode.workspace.getConfiguration().get(DIDACT_OPEN_AT_STARTUP, false);
+}
+
 export function getRegisteredTutorials() : string[] | undefined {
 	const registered : string[] | undefined = context.workspaceState.get(DIDACT_REGISTERED_SETTING);
```

Comment:
not part of this PR but maybe worth to clean up and return the result directly instead of storing it in a const.

Reply:
@lhein  can you create an issue for this so we don't miss it?

Reply:
why not fixing it directly in this one? Its such a small change.